grunt/service-worker.js

```diff
@@ -24,8 +24,12 @@ function writeServiceWorkerFile(staticDirectoryPath, rootUrl, outputPath) {
     handleFetch: true,
     staticFileGlobs: [
       path.join(staticDirectoryPath, '{audio,css,img,js,xslt}', '*'),
-      path.join(staticDirectoryPath, 'fonts', '*.ttf'),
       path.join(staticDirectoryPath, 'manifest.json'),
+
+      // Fonts
+      path.join(__dirname, 'fonts', 'fontawesome-webfont.woff2'),
+      path.join(__dirname, 'fonts', 'enketo-icons-v2.woff'),
+      path.join(__dirname, 'fonts', 'Noto*.ttf'),
```

Comment:
Won't this cache the italics font? The `less` change below removes our need for those, so let's not precache them.

Reply:
Yeah that's right, nice catch